### ISSUE TITLE
linter: js plugin of unocss eslint config is not supported

### ISSUE THREAD

**Issue by SamuNatsu** [oxc-project/oxc#25341]
### What version of Oxlint are you using?

1.77.0

### What command did you run?

oxlint

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```ts
import { defineConfig } from "oxlint";

// https://oxc.rs/docs/guide/usage/linter/config-file-reference.html
export default defineConfig({
  env: {
    browser: true,
    builtin: true,
    vue: true,
  },
  jsPlugins: ["@unocss/eslint-config/flat"],
  plugins: ["eslint", "oxc", "typescript", "unicorn", "vue"],
});
```

### What happened?

Throws error:

```txt
Failed to parse oxlint configuration file.

  × Failed to load JS plugin: @unocss/eslint-config/flat
  │   TypeError: Cannot use 'in' operator to search for 'createOnce' in warn
  │     at registerPlugin (file:///***/node_modules/.pnpm/oxlint@1.77.0/node_modules/oxlint/dist/lint.js:17830:51)
  │     at loadPlugin (file:///***/node_modules/.pnpm/oxlint@1.77.0/node_modules/oxlint/dist/lint.js:17768:51)
```

**Comment by camc314** [comment]
Looks to be configured incorrectly. Take a look at https://github.com/camc314/oxlint-repros/commit/6749a8a179179423c1935539de51fea3c4bbf5d0 for a demo - thanks

**Comment by SamuNatsu** [comment]
It does not work on vue SFC template `class` fields.

Seems the templates are waiting for support.

#15761

**Comment by camc314** [comment]
Yes - vue support is a separate issue, but I thought the focus of this issue is the unocss plugin not working?

**Comment by SamuNatsu** [comment]
It may work, but not applicable to my project. I'm using vue SFC only, no JSX/TSX.
